content/automate/ManageIQ/System/Event/EmsEvent/Hawkular.class/hawkular_datasource.ok.yaml

```diff
@@ -0,0 +1,14 @@
+---
+object_type: instance
+version: 1.0
+object:
+  attributes:
+    display_name: Datasource Success
```

Comment:
what Datasource Success means? what kind of event is this? I think I'm missing a verb here :)